airbyte-integrations/connectors/source-google-analytics-v4/source_google_analytics_v4/spec.json

```diff
@@ -0,0 +1,41 @@
+{
+  "documentationUrl": "https://docs.airbyte.io/integrations/sources/google-analytics-v4",
+  "connectionSpecification": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Google Analytics V4 Spec",
+    "type": "object",
+    "required": ["credentials_json", "view_id", "start_date"],
+    "additionalProperties": false,
+    "properties": {
+      "credentials_json": {
+        "type": "string",
+        "title": "Credentials JSON",
+        "description": "The contents of the JSON service account key. Check out the <a href=\"https://docs.airbyte.io/integrations/sources/googleanalytics\">docs</a> if you need help generating this key.",
+        "airbyte_secret": true
+      },
+      "view_id": {
+        "type": "string",
+        "title": "View ID",
+        "description": "The ID for the Google Analytics View you want to fetch data from. This can be found from the <a href=\"https://ga-dev-tools.appspot.com/account-explorer/\">Google Analytics Account Explorer</a>.",
+        "airbyte_secret": true
+      },
+      "start_date": {
```

Comment:
PLease check is docs what is the oldest possible start date can be?

Reply:
Docs didn't provide that info: https://developers.google.com/analytics/devguides/reporting/core/v4/rest/v4/DateRange

Reply:
Could you please manually test it and put date like 3 years ago. (I remember that we do not have so old data)

Reply:
Tested manualy with `"start_date": "2017-02-11",  "window_in_days": 1456` - tests passed